cmd/blockchaincmd/add_validator.go

```diff
@@ -122,8 +125,9 @@ Testnet or Mainnet.`,
 	cmd.Flags().BoolVar(&waitForTxAcceptance, "wait-for-tx-acceptance", true, "(for Subnets, not L1s) just issue the add validator tx, without waiting for its acceptance")
 	cmd.Flags().Uint16Var(&delegationFee, "delegation-fee", 100, "(PoS only) delegation fee (in bips)")
 	cmd.Flags().StringVar(&subnetIDstr, "subnet-id", "", "subnet ID (only if blockchain name is not provided)")
-	cmd.Flags().StringVar(&validatorManagerOwnerAddress, "validator-manager-owner", "", "validator manager owner address (only if blockchain name is not provided)")
 	cmd.Flags().Uint64Var(&weight, validatorWeightFlag, uint64(constants.DefaultStakeWeight), "set the weight of the validator")
+	cmd.Flags().StringVar(&validatorManagerOwner, "validator-manager-owner", "", "force using this address to issue transactions to the validator manager")
```

Comment:
we technically dont even need to get validator manager using flag anymore. Currently we only get validator manager from on chain if user doesn't provide argument to add validator. But I think its better if we just do this for all cases -> meaning even if user provides argument to add validator. This means that we don't even need `validator-manager-owner ` flag anymore in add_validator, same goes for remove_validator

Reply:
same goes for change_weight

Reply:
The flag is to be used on special situations:
- the user has a multisig setup for the init tx. in this case the multisig (owner) contract does not even have to have balance. the init tx is sent and paid from another account so the address change needs to be considered by CLI
- the user uses a different account for init than for completion tx. particullary, anyone can send the completion tx. same idea, to enable the user to specify which account to use